content/blog/2022/12/kubewarden-during-2022.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Another year of Kubewarden
+authors:
+- Flavio Castelli
+date: 2022-12-28
+---
+
+With 2022 being basically over, it's time to look back at what happened within
+the Kubewarden project during the last twelve months.
+
+## The 1.0 release, CNCF inclusion
+
+I think the major achievements of the project have been reaching the maturity
+state, which happened with its 1.0 release during the month of July, and getting
+[accepted inside of the CNCF](/blog/2022/06/cncf-sandbox-inclusion/).
+
+The CNCF inclusion lead to an exciting integration with
+[ArtifactHub](/blog/2022/07/artifact-hub-supports-kubewarden/).
+This proved to be a great improvement not just for policy authors, but also
+for the all Kubewarden's users who can now find all the existing policies
+on the [familiar ArtifactHub interface](https://artifacthub.io/packages/search?kind=13&sort=relevance&page=1).
+
+By being part of CNCF, Kubewarden got also access to another CNCF project:
+[CLOMonitor](https://clomonitor.io/projects/cncf/kubewarden).
+Thanks to CLOMonitor, Kubewarden's users can have a clear picture of the health
+of the project and assess its development and security practices.
```

Comment:
```suggestion
and can assess its development and security practices.
```

Reply:
I think there was some hiccup with GH: this suggestion was repeated a couple of times. I've accepted the 1st one